apps/apps.c

```diff
@@ -238,11 +238,13 @@ int ctx_set_verify_locations(SSL_CTX *ctx, const char *CAfile,
 int ctx_set_ctlog_list_file(SSL_CTX *ctx, const char *path)
 {
     if (path == NULL) {
-        if (SSL_CTX_set_default_ctlog_list_file(ctx) <= 0) {
-            BIO_puts(bio_err, "Failed to load default Certificate Transparency "
-                     "log list\n");
-        }
-        return 1; /* Do not treat failure to load the default as an error */
+        /*
+         * Try and load the log list from the default location.
+         * We don't treat it as an error if this fails, since the user didn't
+         * explicitly request a log list be loaded and might not need it either.
+         */
+        SSL_CTX_set_default_ctlog_list_file(ctx);
```

Comment:
I'll create a new PR that adds __owur to all of the CT-related boolean functions.

Failure to load a log is reported as an error (in crypto/ct/ct_log.c), but SSL_CTX_set_default_ctlog_list_file() swallows those errors, which is consistent with the behaviour of SSL_CTX_set_default_verify_dir() and SSL_CTX_set_default_verify_file(). I'd be happy to stop it swallowing errors and suppress them here instead though. It arguably makes most sense to suppress them in s_client, depending on whether -requestct or -requirect were used.

I've checked and this appears to only ever return 1 or 0. While I'm adding __owur, I'll check for anywhere the boolean functions are called and swap "x <= 0" for "!x".